pkg/kubelet/status_manager.go

```diff
@@ -104,7 +104,22 @@ func (s *statusManager) SyncBatch() {
 			podFullName := kubecontainer.GetPodFullName(pod)
 			status := syncRequest.status
 			glog.V(3).Infof("Syncing status for %s", podFullName)
-			_, err := s.kubeClient.Pods(pod.Namespace).UpdateStatus(pod.Name, &status)
+
+			var err error
+			statusPod := &api.Pod{
+				ObjectMeta: pod.ObjectMeta,
+			}
+			// TODO: make me easier to express from client code
+			// TODO: resource version on a pod should be saved when dealing with mirror pods so that
+			// subsequent updates don't require a GET.
```

Comment:
I don't quite understand what makes mirror pods different from regular pods in this case...

Also, do we always want to overwrite the status even though kubelet was syncing to an older pod spec?